Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,103 @@
+
+
+### Design Voltage 
+From the information in Auto Light Wiring diagram, a method to detect the output of the relay can be created. The design shown in Figure 2 is to splice and add a voltage divider to connect to the microcontroller. This process uses the 12 V potential that is sent to the lamp to illuminate the light to trigger a reaction on the microcontroller. 
+
+The design behind the voltage divider is to reduce the auto light voltage to no more than 5 V and have a current of less than 1% of the current to the light. This constraint on current is to prevent any exceedance in the limits of the original hardware and wire ratings that could cause overload or failure on the car. The current through the light is 4.79 A which makes the limit to the voltage divider 47.9 mA. 40 mA is decided to be a limit for the safety of the devices added and not exceed the constraint. Through Ohm's law, the total resistance is found to be approximately 375 Ohms assuming that the input voltage could vary from 10 to 15 V. This achieves a maximum of 5 V from a 15 V input between the two series resistors of 250 Ohms and 125 Ohms respectively. The average active voltage applied should be approximately 12 V. Thus, the limit should not be exceeded. A 40 mA fuse will be installed to protect against any extreme voltage spikes outside of the 10 to 15 V range that would cause overcurrents. The LTV-817 optoisolator was chosen based on the threshold voltage of 1.2 V, and the maximum current constraint of 40 mA is less than the limit for the LTV-817 at 50 mA.
+
+However, the design with only a voltage divider has a potential risk due to the chance of a voltage spike reaching the microcontroller and interference from other signals. This design shows the vulnerability to signal interferece through the shared ground terminal and the ability for the current to backfeed onto another divider. The voltage divider prevents the chance of the signal voltage to exceed the limit of 5 V due to a spike in the input voltage on the lamp node. The design was modified to include optoisolators to remidy the hazards of both constraints as shown in Figure 2. Optoisolators operate on a threshold voltage and are not reliant on current. This means that it can operate on very small currents as long as the voltage meets the minimum requirement. The optoisolator will take the reduced voltage from the Nissan Vehicle and convert it into a light signal. Then a photosensor will detect the light signal and allow the 5 V to pass through the pin of the microcontroller. The LED that is used to trigger the photosensor also prevents any backfeed current from false triggering a signal. This isolates both systems so that the voltage to the lights does not have a direct path to the signal input. With the microcontroller being used for the 5 V supply in the signal, there should not be a voltage spike to the controller since it would have to be generated by the controller.
+
+## Analysis
```

Comment:
What are you analyzing here? You need to state the constraints and specs in a precise, measurable way so that here it is clear what you are analyzing.